integration-cli/docker_cli_run_test.go

```diff
@@ -2538,3 +2538,7 @@ func (s *DockerSuite) TestRunNetworkFilesBindMount(c *check.C) {
 		c.Fatalf("expected resolv.conf be: %q, but was: %q", expected, actual)
 	}
 }
+
+func (s *DockerSuite) TestRunCapAddSYSTIME(c *check.C) {
+	dockerCmd(c, "run", "--cap-drop=ALL", "--cap-add=SYS_TIME", "busybox", "sh", "-c", "date --set `date '+%H:%M:%S'` > /dev/null", "true")
```

Comment:
@tiborvass `dockerCmd` doesn't return an error, it calls `c.Fatal` with the error and the output on it's own.